### ISSUE TITLE
Can't have urls for my products

### ISSUE THREAD

**Issue by echantigny** [craftcms/commerce#369]
Database driver & version: MySQL 5.7.21
Craft CMS: Craft Solo 3.0.10.2
Craft Commerce: 2.0.0-beta.5
Redactor: 2.1.0

For some reason, it seems that I can't have urls for my products. The checkbox for it doesn't even exist in the Product Type section. Is this an issue or I'm missing something? I've been looking at this page to confirm my thought: https://github.com/craftcms/commerce-docs/blob/v2/en/product-types.md

![2018-06-07 10_29_43-clothing - great boardgames](https://user-images.githubusercontent.com/4933333/41114882-b60aa8e4-6a53-11e8-81dd-55473fa08e1e.png)

**Comment by echantigny** [comment]
Oaky, this is not an actual bug, but it was confusing.  A product doesn't have a url if you only have the title field in there.  Adding a description field made the url active.

The documentation is also false now though, and it also makes the demo products a bit confusing.

**Comment by echantigny** [comment]
Hi @lukeholder ,

From what I remember, it was a clean install:

1- Install Craft 3
2- Install Craft Commerce 2
3- Look at the site (all default products and templates at this point)

When I got to the products listing, nothing was clickable.  I added a field (description wysiwyg) and then all the products became clickable.  All this with the basic templates that comes with the plugin.

If needed, I can definitely try a new install and see if I can replicate that.  I played around a lot with it since then.
